### PR TITLE
edits to AGGREGATE function variables

### PR DESCRIPTION
**Pull request by lenapivotal**
Made some small changes, sfunc -> statefunc where I saw variable name in the xml. 
Also, removed stype reference in the text. 
But I'm not 100% certain that's what you're looking for David.   

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
 <topic id="topic1"><title id="bm20941">CREATE AGGREGATE</title><body><p id="sql_command_desc">Defines a new aggregate function.</p><section id="section2"
       ><title>Synopsis</title><varname>argname</varname><codeblock id="sql_command_synopsis">CREATE AGGREGATE <varname>name</varname> ( [ <varname>argmode</varname> ] [  ] <varname>arg_data_type</varname> [ , ... ] ) (
-    SFUNC = <varname>sfunc</varname>,
+    STATEFUNC = <varname>sfunc</varname>,
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml]: The uppercase instances here are part of the SQL syntax,so we can't change them.  Only the lower-case instances (tagged as varname style) can be changed.

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml [hunk 2]
```diff
@@ -107,35 +107,37 @@
...
-        the segments to generate partial aggregate results, and
-            <codeph><varname>combinefunc</varname></codeph> is executed on the master to aggregate
-        the partial results from segments. If single-level aggregation is performed, all the rows
-        are sent to the master and <codeph><varname>sfunc</varname></codeph> is applied to the
-        rows.</p><p>Single-level aggregation and two-level aggregation are equivalent execution strategies. Either
+        two-level execution is performed, the state transition function
+            <codeph><varname>statefunc</varname></codeph> is executed on the segments to generate
+        partial aggregate results, and <codeph><varname>combinefunc</varname></codeph> is executed
+        on the master to aggregate the partial results from segments. If single-level aggregation is
+        performed, all the rows are sent to the master and the state transition function
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml]: Probably don't need to repeat "state transition function" here.

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml [hunk 3]
```diff
@@ -107,35 +107,37 @@
...
-          <codeph>combinefunc</codeph> and <codeph>sfunc</codeph>, you must ensure that the
-        invocation of <codeph>sfunc</codeph> on the segment instances followed by
-          <codeph>combinefunc</codeph> on the master produce the same result as single-level
-        aggregation that sends all the rows to the master and then applies only the
-          <codeph>sfunc</codeph> to the rows.</p><p>An aggregate function can
+          <codeph>combinefunc</codeph> and <codeph>statefunc</codeph>, you must ensure that the
+        invocation of the state transition function <codeph><varname>statefunc</varname></codeph> on
+        the segment instances followed by <codeph>combinefunc</codeph> on the master produce the
+        same result as single-level aggregation that sends all the rows to the master and then
+        applies only the state transition function <codeph><varname>statefunc</varname></codeph>  to
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml]: Can also remove "state transition function" here as it's already called out earlier in the sentence.

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml [hunk 4]
```diff
@@ -231,16 +233,17 @@
...
-          <pt><varname>sfunc</varname></pt>
+          <pt><varname>statefunc</varname></pt>
           <pd>The name of the state transition function to be called for each input row. For a
-            normal N-argument aggregate function, the <varname>sfunc</varname> must take N+1
-            arguments, the first being of type <varname>state_data_type</varname> and the rest
-            matching the declared input data types of the aggregate. The function must return a
-            value of type <varname>state_data_type</varname>. This function takes the current state
-            value and the current input data values, and returns the next state value.</pd>
+            normal N-argument aggregate function,  the state transition function
+                <codeph><varname>statefunc</varname></codeph> must take N+1 arguments, the first
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml]: Again, after the initial definition here, it's not necessary to repeat the full name "state transition function".

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml [hunk 5]
```diff
@@ -411,7 +414,7 @@ the sum of two columns. </p><p>Before creating the aggregate function, create tw
    IMMUTABLE
    RETURNS NULL ON NULL INPUT;</codeblock><p>This <codeph>CREATE AGGREGATE</codeph> command creates the aggregate function
 that adds two columns. </p><codeblock>CREATE AGGREGATE agg_prefunc(numeric, numeric) (
-   SFUNC = mysfunc_accum,
+   STATEFUNC = mysfunc_accum,
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml]: Uppercase syntax needs to remain unchanged here.

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml [hunk 6]
```diff
@@ -395,10 +398,10 @@ ordered aggregate, using the syntax:
 <codeph>COMBINEFUNC</codeph>.</p>
 
 </section><section id="section7"><title>Example</title><p>The following simple example creates an aggregate function that computes
-the sum of two columns. </p><p>Before creating the aggregate function, create two functions that are used as the
-            <codeph><varname>sfunc</varname></codeph> and
-          <codeph><varname>combinefunc</varname></codeph> functions of the aggregate function. </p><p>This function is specified as the <codeph><varname>sfunc</varname></codeph> function in the
-        aggregate function.</p><codeblock>CREATE FUNCTION mysfunc_accum(numeric, numeric, numeric) 
+the sum of two columns. </p><p>Before creating the aggregate function, create two functions that are used as the state
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml]: "state transition function" is awkward in this sentence, as it makes it sound like two functions form the state transition function. Best to just remove it I think.

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml (hunk 1 above)
Comment (lenapivotal) [reply gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml]: fixed in new commit

File: gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml [hunk 7]
```diff
@@ -411,7 +412,7 @@ the sum of two columns. </p><p>Before creating the aggregate function, create tw
    IMMUTABLE
    RETURNS NULL ON NULL INPUT;</codeblock><p>This <codeph>CREATE AGGREGATE</codeph> command creates the aggregate function
 that adds two columns. </p><codeblock>CREATE AGGREGATE agg_prefunc(numeric, numeric) (
-   SFUNC = mysfunc_accum,
+   STATEFUNC = mysfunc_accum,
```
Comment (hlinnaka) [inline gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml]: `SFUNC' is a keyword, the example won't work if you change it.

Comment (lenapivotal) [reply gpdb-doc/dita/ref_guide/sql_commands/CREATE_AGGREGATE.xml]: Thank you, I have updated the branch.

**Pull request merged**